models/inventory-entry/src/inventory-entry-draft/presets/sample-data-b2c-lifestyle/index.ts

```diff
@@ -6,11 +6,13 @@ import skuADPC7 from './sku-ADPC7';
 import skuALC01 from './sku-ALC01';
 import skuAMR09 from './sku-AMR09';
 import skuARG56 from './sku-ARG56';
+import skuARG56Channel from './sku-ARG56-channel';
```

Comment:
It seems we're duplicating some presets instead of updating them with the new values.
This will imply that, for instance, `skuARG56` would be deprecated, consumers would need to update their code to use the new `skuARG56Channel` and we would need to remove the former later in time.

Wouldn't it make more sense to update the existing presets?

I think this is related to a conversation I had last week with @jaikamat about the original project data used in the presets generation repository to have changed, which is a maintainability problem.

Perhaps we can have a chat about this.
What do you think?

Reply:
Thanks for the explanation. I think I understand now. 🙇 

> For the B2C dataset, there was never an original project for the data. It was generated from spreadsheets. I can confirm, the data is constantly shifting based on requests from coFe.
Let's talk about this a bit more in a meeting.